src/components/Profile/Profile.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
```

Comment:
Will this page serve as the homepage of users, I mean as the dashboard? @rwajon

Reply:
yes, when a user goes to their profile, this will be the default page

Reply:
@rwajon great